spec/board_spec.rb

```diff
@@ -0,0 +1,231 @@
+        it 'returns "O" when, given a board where O claims middle column' do
+            subject.make_move(0, 1, "O")
+            subject.make_move(1, 1, "O")
+            subject.make_move(2, 1, "O")
+
+            expect(subject.find_winner).to eql('O')
+        end
+        it 'returns "O" when, given a board where O claims right column' do
+            subject.make_move(0, 2, "O")
+            subject.make_move(1, 2, "O")
+            subject.make_move(2, 2, "O")
+
+            expect(subject.find_winner).to eql('O')
+        end
+        it 'returns "X" when, given a board where X claims the right-left diagonal' do
+            subject.make_move(0, 0, "X")
+            subject.make_move(1, 1, "X")
+            subject.make_move(2, 2, "X")
+
+            expect(subject.find_winner).to eql('X')
+        end
+        it 'returns "X" when, given a board where X claims the left-right diagonal' do
+            subject.make_move(0, 2, "X")
+            subject.make_move(1, 1, "X")
+            subject.make_move(2, 0, "X")
+
+            expect(subject.find_winner).to eql('X')
+        end
+        it 'returns "O" when, given a board where O claims the right-left diagonal' do
+            subject.make_move(0, 0, "O")
+            subject.make_move(1, 1, "O")
+            subject.make_move(2, 2, "O")
+
+            expect(subject.find_winner).to eql('O')
+        end
+        it 'returns "O" when, given a board where O claims the left-right diagonal' do
+            subject.make_move(0, 2, "O")
+            subject.make_move(1, 1, "O")
+            subject.make_move(2, 0, "O")
+
+            expect(subject.find_winner).to eql('O')
+        end
+        it 'returns :tie when, given a board where neither X nor O wins' do
+            subject.make_move(0, 0, "X")
+            subject.make_move(0, 1, "O")
+            subject.make_move(0, 2, "X")
+            subject.make_move(1, 0, "X")
+            subject.make_move(1, 1, "O")
+            subject.make_move(1, 2, "O")
+            subject.make_move(2, 0, "O")
+            subject.make_move(2, 1, "X")
+            subject.make_move(2, 2, "X")
+
+            expect(subject.find_winner).to eql(:tie)
+        end
+    end
+
+    describe "#game_over?" do
+        it "returns false, when no players have placed letters" do
+            subject = Board.new
+            expect(subject.game_over?).to eql(false)
+        end
+
+        it "returns true when the board is full" do
+            subject = Board.new
+            subject.make_move(0, 0, "X")
+            subject.make_move(0, 1, "X")
+            subject.make_move(0, 2, "X")
+            subject.make_move(1, 0, "X")
+            subject.make_move(1, 1, "X")
+            subject.make_move(1, 2, "X")
+            subject.make_move(2, 0, "X")
+            subject.make_move(2, 1, "X")
+            subject.make_move(2, 2, "X")
+
+            expect(subject.game_over?).to eql(true)
+        end
+        # pending "returns true if either player has won"
```

Comment:
Okay, it looks like it is a WIP, I don't think the `#` in front of `pending` is useful. RSpec will recognize `pending` and will show in your test output/CI that you have a pending test, that way you can remember to resolve it later.